acpi.adoc

```diff
@@ -3,31 +3,20 @@
 
 The Advanced Configuration and Power Interface specification provides the OS-centric view of system configuration, various hardware resources, events and power management.
 
-This section defines mandatory and optional ACPI requirements on top of the ones in the ACPI cite:[ACPI] and UEFI cite:[UEFI] specifications. Additional non-normative guidance may be found in the <<acpi-guidance, appendix>>.
+This section defines the BRS-I mandatory and optional ACPI requirements on top of cite:[ACPI] and cite:[UEFI]. Additional non-normative guidance may be found in the <<acpi-guidance, appendix>>.
 
 A compliant system must:
 
 * [[acpi-64bit-clean]]Be 64-bits clean.
 ** RSDT must not be implemented, with RsdtAddress in RSDP set to 0.
 ** 32-bit address fields must be 0.
 ** <<acpi-guidance-64bit-clean, See additional guidance>>.
-* <<acpi-hw-reduced, Implement a hardware-reduced ACPI model>>.
-* Implement a firmware-first APEI model. (cite:[ACPI] Section 18.4)
-
-[[acpi-hw-reduced]]
-=== Hardware-Reduced ACPI Model
-
-Compliant RISC-V systems only implement the HW-Reduced ACPI Model cite:[ACPI] (Section 4.1).
-This means the hardware portion of cite:[ACPI] (Section 4) is not required or
-supported. All functionality is instead provided through equivalent
-software-defined interfaces and the complexity in supporting ACPI is reduced.
-
-The following additional requirements apply:
-
-* No FACS table.
-* For the ACPI Platform Error Interfaces (APEI) support:
-** There is 1 event for non-fatal error signaling (cite:[ACPI] Section 18.3.2.7.2)
-** There is 1 NMI-equivalent signal for use in fatal errors (cite:[ACPI] Section 18)
+* [[acpi-hw-reduced]]Implement the HW-Reduced ACPI Mode.
+** No FACS table.
+** <<acpi-guidance-hw-reduced, See additional guidance>>.
+* Implement a firmware-first ACPI Platform Error Interface (APEI) model (cite:[ACPI] Section 18.4).
```

Comment:
Does this imply this is the only model? Or that something is exposed as opt-in for firmware first.

Reply:
Are you referring to the APEI portion? I just moved the text, but we can take the APEI requirements out for now if we need to dig into the details here (e.g. my personal opinion is that for BRS-I we mandate APEI, but there may be more work on the APEI side - but that's not really a BRS task by itself)

Reply:
Ya, I was. I know you were moving things, but I did want to ask this question about the implications. I was mainly talking about 'firrmware-first APEI' as the only option. If that's the case, that's a discussion to be had.  I can put it on the agenda for the next meeting.

Reply:
Ok, let's put that on the agenda. I raised https://github.com/riscv-non-isa/riscv-brs/issues/71 so we don't lose track of it. I'll merge this pull request meanwhile.